pkg/cli-runtime/testing/client.go

```diff
@@ -50,7 +59,132 @@ func NewFakeCliClient(c crclient.Client) cli.Client {
 	}
 }
 
+// RESTClient returns a REST client from TestFactory
+func (c *fakeclient) RESTClient() (*rest.RESTClient, error) {
+	// Swap out the HTTP client out of the client with the fake's version.
+	fakeClient := c.Clients.(*fake.RESTClient)
+	restClient, err := rest.RESTClientFor(c.ClientConfigVal)
+	if err != nil {
+		panic(err)
+	}
+	restClient.Client = fakeClient.Client
+	return restClient, nil
+}
+func (c *fakeclient) RESTMapper() meta.RESTMapper {
+	return testRESTMapper()
+}
+func (c *fakeclient) ToRESTMapper() (meta.RESTMapper, error) {
+	return c.RESTMapper(), nil
```

Comment:
why are there 2 func `ToRESTMapper` and `RESTMapper`?

Reply:
Removed Restmapper